haskell/sorting-in-reverse-order.md

```diff
@@ -0,0 +1,24 @@
+# Sorting in reverse order
+
+`Data.Ord` contains a newtype, `Down`, which can wrap any type that has an
+`Ord` instance, in order to provide a reversed `Ord` instance for that type.
+
+For example:
+
+```hs
```

Comment:
hs works, as does rb, etc. At least in Pandoc and GitHub, but I agree -- since this is a ghci session, it should be unhighlighted.